cluster/gce/manifests/kube-addon-manager.yaml

```diff
@@ -8,11 +8,19 @@ metadata:
   labels:
     component: kube-addon-manager
 spec:
+  securityContext:
+    runAsUser: {{runAsUser}}
```

Comment:
Why is this parameterized? Just pick a uid.

Reply:
The key idea is to control distribution of UIDs and GIDs centrally, and being able to write tests to ensure that these IDs are unique.

Reply:
do we definitely need `UID != GID`?

Reply:
Nope, the values will be the same. But if we need to for some reason in the future we will be able to control that from the centralized config without having to make a change like this again.